### DISCUSSION TITLE
Results on only one of the dataset, e.g. VID.

### DISCUSSION THREAD

**Discussion by willtwr** [STVIR/pysot#67] [Reproduce]
Has anyone tried to train with only one of the dataset (e.g. only VID) and evaluate the model? 
It is much appreciated if anyone who did such experiment can share the result.
Thanks!

**Comment by xi-mao** [comment]
I have tried to train  only using vid，but when I test ，I found the tracker cann't track  the object at all。I don't know why。 I think it probably something wrong in my traning data。 But I generate traning data  as the code completely。

**Comment by NineBall9** [comment]
I have trained only using vid, the result is 0.26, the difference is too large.
![image](https://user-images.githubusercontent.com/27496052/60167646-0faa8480-9836-11e9-9b09-610d0a9d69f4.png)

**Comment by xi-mao** [comment]
I have trained using vid and ytbb. The auc on otb100 is 0.54。 I also trained only using vid。 but i havn't test it。

**Comment by CXSunshine** [comment]
> I have trained using vid and ytbb. The auc on otb100 is 0.54。 I also trained only using vid。 but i havn't test it。

How did you download the ytbb datasets?It is too big and need to google.

**Comment by willtwr** [comment]
Since it has more number of iterations when using 4 training sets compared to 1 or 2 of the datasets, perhaps training the tracker for more iterations might help?

**Comment by NineBall9** [comment]
> I have trained using vid and ytbb. The auc on otb100 is 0.54。 I also trained only using vid。 but i havn't test it。
@xi-mao Could you test the model trained only on vid , and tell me the result on VOT2018? Thank you!

**Comment by zhanght021** [comment]
> > I have trained using vid and ytbb. The auc on otb100 is 0.54。 I also trained only using vid。 but i havn't test it。
> > @xi-mao Could you test the model trained only on vid , and tell me the result on VOT2018? Thank you!

@NineBall9  I have trained using vid coco and GOT-10k, the eao in VOT 2016 is 0.383, but eao in VOT2018 is only 0.28-0.30, the eao in VOT2019 is only only 0.245, the backbone i used is  ### **alexnet**###

**Reply by cunjif** [reply]
Please, can tell me your hardware configs ? 8GPUS?

**Comment by lzx1413** [comment]
@zhanght021  https://github.com/researchmm/SiamDW/issues/20#issuecomment-502104820
pay attention to 3,4,5

**Comment by Ling-wei** [comment]
I trained SiamRPN++(resnet-50) for 20 epochs, using **COCO、VID、DET**, and only got EAO **0.31** for VOT2018 and **0.25** for VOT2019 .  The best checkpoint comes **checkpoint_10.pth**, and the 10-20 Epochs training for **ResNet&SiamRPN++** doesn't work.

**Comment by charlesyann** [comment]
> I trained SiamRPN++(resnet-50) for 20 epochs, using **COCO、VID、DET**, and only got EAO **0.31** for VOT2018 and **0.25** for VOT2019 . The best checkpoint comes **checkpoint_10.pth**, and the 10-20 Epochs training for **ResNet&SiamRPN++** doesn't work.

get the same result with you, it seems youtube-bb dataset is necessarily needed. @Ling-wei

**Comment by NineBall9** [comment]
> I have tried to train only using vid，but when I test ，I found the tracker cann't track the object at all。I don't know why。 I think it probably something wrong in my traning data。 But I generate traning data as the code completely。

@xi-mao ，how long it costs to train with data only VID, backbone resnet50, epoch 20 and 8 gpus?

**Comment by xi-mao** [comment]
@NineBall9  Traning SiamRPN with vid  toke me one and half days using two GPUs

**Comment by willtwr** [comment]
Has anyone tried to use all 4 datasets to reproduce the results?

**Comment by gongbudaizhe** [comment]
Training with only VID won't work. You need at least VID + COCO to get reasonable results. 
For now, the best results I can get using VID + COCO + YTB on OTB100 is 0.681 without searching hyperparameters. The most stable results are 0.676.

Using the same hyper-parameters with models at different epochs, the best EAO on VOT2018 is 0.317.
